.github/workflows/test-backends.yml

```diff
@@ -0,0 +1,91 @@
+name: "Build backends with an updated Colvars module"
+
+on: [push, pull_request]
```

Comment:
@HubLot As far as I understood, the standard `pull_request` would run the workflows entirely from the PR's head, and since this can be located anywhere, the workflows wouldn't have access to the repository secrets. And unfortunately we need those as long as VMD and NAMD remain with their restrictive license and their repositories need to remain private.

Reply:
you're right, I'm trying to see if there is a solution to that but with little hope. 
It will work for PRs with a branch coming from the official 'colvars' depot but not from a fork.